src/feeds.js

```diff
@@ -308,64 +306,86 @@ feeds.fetchEventsFromCalendar_ = function(feed, callback) {
     }
     _gaq.push(['_trackEvent', 'Fetch', 'Events']);
 
-    $.ajax(feedUrl, {
-      headers: {'Authorization': 'Bearer ' + authToken},
-      success: (function(feed) {
-        return function(data) {
-          background.log('Received events, now parsing.', feed.title);
-          var events = [];
-          for (var i = 0; i < data.items.length; i++) {
-            var eventEntry = data.items[i];
-            var start = utils.fromIso8601(eventEntry.start.dateTime || eventEntry.start.date);
-            var end = utils.fromIso8601(eventEntry.end.dateTime || eventEntry.end.date);
-
-            var responseStatus = '';
-            if (eventEntry.attendees) {
-              for (var attendeeId in eventEntry.attendees) {
-                var attendee = eventEntry.attendees[attendeeId];
-                if (attendee.self) {
-                  // Of all attendees, only look at the entry for this user (self).
-                  responseStatus = attendee.responseStatus;
-                  break;
-                }
+    var fromDate = moment();
+    var fetchEventsRecursively = function(days) {
+      background.log('fetchEventsRecursively()', feed.title);
```

Comment:
Maybe there is a cleaner was to log here, assuming we want log output?

Reply:
You can skip the logging; I’d added it way back to confirm things were working as expected.

BTW, a simpler approach would have worked too: if (nothing in the next 14 days), grab the next 100 days (all at once.)